generators/client/templates/angular/src/main/webapp/app/app.main.ts.ejs

```diff
@@ -16,6 +16,7 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+import './polyfills';
```

Comment:
any specific reason to move to this approach or can we follow cli generated code to add it to `tsconfig.app.json` like 
```"files": ["src/main.ts", "src/polyfills.ts"],```

Reply:
I just thought it reduce entry points, I don't see why we put the polyfills apart...

Reply:
can you confirm that it still produces a different bundle for polyfills? If not, then, it's a deviation from current behavior and also wouldn't recommend that.

Reply:
@wmarques, can you confirm the behavior?

Reply:
It doesn't produces a different bundle but the polyfill would still be loaded so for me that's not really important, polyfills are always loaded, we don't do differential loading like angular CLI does.